src/main/java/godaddy/registry/jtoolkit2/se/fee/DomainCreateFeeCommandExtension.java

```diff
@@ -30,6 +30,7 @@
     private static final String FIELD_IDENTIFIER = "<<field>>";
     private BigDecimal registrationFee;
     private String currency;
+    private boolean descriptionRequired = true;
```

Comment:
```suggestion
    private final boolean descriptionRequired;
```
Please refer to my other comment.

Reply:
Done